### DISCUSSION TITLE
已购买2年版的专业版，无法授权其他设备，ios手机无法登录。账户connie0125@126.com

### DISCUSSION THREAD

**Discussion by xuanxuan0125** [getlantern/lantern-forum-cn#231] [Q&A]
点击授权设备，邮箱无法收到验收码。IOS应用商店无法下载。
![1710161491952](https://github.com/getlantern/lantern-forum-cn/assets/163001096/6a669979-124f-4b9e-a1b7-6ee0c4103001)

**Comment by MaxMKMax** [comment]
@xuanxuan0125 好的。我将帮助您请求团队为您手动升级到专业版。同时，您能分享一下付款收据吗？谢谢。

**Comment by MaxMKMax** [comment]
@xuanxuan0125 你好! IOS下载可以通过这里的链接。您可以按照说明下载适用于您其他设备的 ios 应用程序。谢谢。
https://lantern.io/zh-Hans/faq

**Reply by MaxMKMax** [reply]
@xuanxuan0125 晚上好! 我已经通知负责人处理您的请求。由于其他用户有很多请求票，请给我们 1 或 2 天的时间。我们将尽快处理您的问题。再次感谢您对蓝灯笼的支持。我们很高兴拥有像您这样的宝贵用户。谢谢!

**Comment by xuanxuan0125** [comment]
附件为账单详情，请查收。

---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年03月11日 22:46 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | xuanxuan0125 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] ***@***.*** (Discussion #231) |

@xuanxuan0125 晚上好! 我已经通知负责人处理您的请求。由于其他用户有很多请求票，请给我们 1 或 2 天的时间。我们将尽快处理您的问题。再次感谢您对蓝灯笼的支持。我们很高兴拥有像您这样的宝贵用户。谢谢!

**Reply by MaxMKMax** [reply]
@xuanxuan0125 您好。当前版本仍然存在登录问题。请给我们几天的时间来解决，我们会尽快更新您的信息。我们将很快发布最新版本。抱歉让您久等了。稍后我们还将补偿您更长的专业版。

**Reply by MaxMKMax** [reply]
@xuanxuan0125 我们会尽快帮您处理. 请留意邮件. 谢谢!

**Comment by xuanxuan0125** [comment]
目前问题还未解决，如果解决不了。麻烦给我退款吧，谢谢！
















在 2024-03-18 23:53:09，"Mike K" ***@***.***> 写道：

@xuanxuan0125 我们会尽快帮您处理. 请留意邮件. 谢谢!

**Comment by MaxMKMax** [comment]
@xuanxuan0125 您好美女, 上两周已经提交给客服了. 他们应该有回复您邮件并带上授权码. 您检查一下邮箱或者垃圾邮箱有吗? 如果没有您再回来找我. 我今晚在帮您追他们. 答应你两天内能搞定. 非常抱歉啊. 谢谢您等待!

**Comment by xuanxuan0125** [comment]
很久之前收到一个这样的激活码，但使用了之后时间专业版的时间吧变成了1年了，而且还是不能授权其他电脑。










在 2024-04-01 16:47:33，"Mike K" ***@***.***> 写道：

@xuanxuan0125 您好美女, 上两周已经提交给客服了. 他们应该有回复您邮件并带上授权码. 您检查一下邮箱或者垃圾邮箱有吗? 如果没有您再回来找我. 我今晚在帮您追他们. 答应你两天内能搞定. 非常抱歉啊. 谢谢您等待!

**Comment by xuanxuan0125** [comment]
你好

 都一个多月过去了，还是不能授权另一台电脑而且使用期限也没有恢复。能不能给我一个授权码，我直接用来授权另一台电脑。然后日期帮我改成2年的，我买了2年，结果变成1年的了。如果实在不行，就按我使用的时间给我退款吧。




在 2024-04-01 16:47:33，"Mike K" ***@***.***> 写道：

@xuanxuan0125 您好美女, 上两周已经提交给客服了. 他们应该有回复您邮件并带上授权码. 您检查一下邮箱或者垃圾邮箱有吗? 如果没有您再回来找我. 我今晚在帮您追他们. 答应你两天内能搞定. 非常抱歉啊. 谢谢您等待!

**Reply by MaxMKMax** [reply]
@xuanxuan0125 嗯嗯. 可以处理的! 我帮您找看之前的处理记录. 如果没错他们又给您发过邮件了. 可能您是 126.com的邮箱所以收不到. 您放心, 我再帮您解决这个问题. 一个授权码和 2 年.

**Comment by MaxMKMax** [comment]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/51a60660-afe0-4231-b548-33cc2979e20e)

@xuanxuan0125 您好. 您的专业账号是到 2025 7 月的. 请问您是什么时候购买的?

请问您的授权码的问题解决了吗?

支付团队那里需要您提供一下支付单据. 麻烦您了, 谢谢!

**Comment by MaxMKMax** [comment]
@xuanxuan0125 我们的负责人说: 我在专业面板上没有看到任何可以分享的电子邮件链接代码.

可能需要您放一下您要连接的邮箱才能启动. 您可以参考这里怎么授权: https://lantern.io/zh-Hans/faq

**Comment by xuanxuan0125** [comment]
我买的是2两的呀，不应该是2025年的。授权码的问题还是没有解决。邮箱就是这个给你们发邮件的邮箱呀，能不能给我发一个授权到其他电脑的验证码。我现在就是没有办法授权其他电脑。
















在 2024-05-16 15:22:47，"Mike K" ***@***.***> 写道：

image.png (view on web)

@xuanxuan0125 您好. 您的专业账号是到 2025 7 月的. 请问您的授权码的问题解决了吗?

**Reply by MaxMKMax** [reply]
@xuanxuan0125 嗯嗯好的. 我们会在帮您更新时间线. 我再帮您问问看. 刚好今天负责人不在线. 要等到下周一晚上他才会上线了. 不好意思让您久等了.

授权码是需要您亲自去授权您的另一个电脑的. 好像无法直接发授权码之类的操作. 请您参考授权码选项: https://lantern.io/zh-Hans/faq
